shinkai-libs/shinkai-db/src/db/db_jobs.rs

```diff
@@ -833,6 +837,22 @@ impl ShinkaiDB {
             }
         }
 
+        // Delete inbox messages
+        let inbox_hash = InboxName::new(inbox_name.to_string())?.hash_value_first_half();
+        let inbox_key_prefixes = vec![
+            format!("inbox_{}_message_", inbox_hash),
+            format!("inbox_{}_parent_", inbox_hash),
+            format!("inbox_{}_children_", inbox_hash),
```

Comment:
i think there is a bug here. the prefix iterator only works with the specific length that was set up when creating the specific topic e.g. 47 characters

if we use the prefix with 46 or 48 it wont work. i dont remember from the top of my mind if it would show everything (including stuff that doesnt contain the prefix) or none though